docs/source/azure/configuration/containerhosting.rst

```diff
@@ -3,24 +3,18 @@
 Azure Container Hosting
 =======================
 
-The Azure provider can be run in a containerized environment using the official cloud custodian 
-`docker image <https://hub.docker.com/r/cloudcustodian/c7n>`_. The Azure Container Host knows 
-how to handle both periodic and event based policies.
+The Azure Container Host is an alternative execution mode for the cloud custodian azure provider.
+Running the Azure Container Host is done with the `official custodian docker image <https://hub.docker.com/r/cloudcustodian/c7n>`_
+by providing the custom entry point, ``/usr/local/bin/python3 -m c7n_azure.container_host.host``, 
```

Comment:
Rather than providing this level of detail here, I'd say that there is a public docker image and for deployment options see "insert links to the instructions for deploying here". This helps keep it high level and generic for the introduction